internal/generator/generator.go

```diff
@@ -377,6 +377,15 @@ func NewGenerator(
 	}
 
 	// check admin features for resources
+	revisionHistory := CheckAdminFeatureFlag("DEPLOYMENT_REVISION_HISTORY", false)
+	if revisionHistory != "" {
+		rhInt, err := strconv.Atoi(revisionHistory)
+		if err != nil {
+			return nil, fmt.Errorf("revision history does not convert to integer, contact your Lagoon administrator")
```

Comment:
```suggestion
			return nil, fmt.Errorf("provided revision history %s is not a valid integer", revisionHistory)
```

Aren't lagoon administrators the only ones that can set this value?